web/templates/settings_overlay.hbs

```diff
@@ -131,10 +131,10 @@
                     </li>
                     {{/if}}
                     {{#unless is_admin}}
-                    <div class="sidebar-item collapse-settings-btn">
-                        <i id='toggle_collapse_chevron' class='fa fa-angle-double-down'></i>
-                        <p id='toggle_collapse'>{{t "Show more" }}</p>
-                    </div>
+                    <li class="sidebar-item collapse-settings-button">
+                        <i id='toggle_collapse_chevron' class='icon fa fa-angle-double-down'></i>
```

Comment:
The `icon` class shouldn't be added here now.

Reply:
We still need properties from that class. It is used for all other icons above. We can change the name of that class in a different commit to `sidebar-item-icon`

Reply:
I've added 7432aa8ec37c4c86c6ac63d412137c67c4ef1716

Reply:
Ahh, OK, didn't realize it wasn't a new class. Looks good.